python/cudf/dataframe/numerical.py

```diff
@@ -90,9 +95,19 @@ def normalize_binop_value(self, other):
         other_dtype = np.min_scalar_type(other)
         if other_dtype.kind in 'biuf':
             other_dtype = np.promote_types(self.dtype, other_dtype)
-            ary = utils.scalar_broadcast_to(other, shape=len(self),
-                                            dtype=other_dtype)
-            return self.replace(data=Buffer(ary), dtype=ary.dtype)
+            if other_dtype == np.dtype('int16'):
+                other_dtype = np.dtype('int32')
```

Comment:
Do we still need this? I might have mucked up while resolving a merge conflict..

Reply:
I don't think so. That was a workaround for dealing with `int16`, which I think our code can do

Reply:
This should be removed